dist/common/scripts/scylla_util.py

```diff
@@ -99,6 +99,17 @@ def scyllabindir():
 def sysconfdir():
     return str(sysconfdir_p())
 
+def configure_aio_slots():
+    required_aio_slots = str(cpu_count() * 11026 + 65536)
+    aio_max_nr_value = "fs.aio-max-nr = " + required_aio_slots
+    with open("/etc/sysctl.conf", "r+") as file:
+        for line in file:
+            if aio_max_nr_value in line:
```

Comment:
This does not look right. You're checking if that _exact_ line already exists, but what if `fs.aio-max-nr` is already configured in `sysctl.conf`, but with different value?

So I think this line should instead be:

```python
if "fs.aio-max-nr" in line
```